schema/field.go

```diff
@@ -10,10 +10,11 @@ type Field struct {
 	// Kind is the basic type of the field.
 	Kind Kind
 
-	// Nullable indicates whether null values are accepted for the field.
+	// Nullable indicates whether null values are accepted for the field. Key fields CANNOT be nullable.
 	Nullable bool
 
 	// AddressPrefix is the address prefix of the field's kind, currently only used for Bech32AddressKind.
+	// TODO: add validation for valid address prefixes
```

Comment:
Should there be validation of address prefixes? It seems an address prefix should be all lowercase ASCII letters and relatively short, however when looking at the bech32 code, it seems there are no such constraints. Should we leave without validation or specify something? At a minimum no null characters should be allowed

Reply:
Any thoughts here? Maybe we'll deal with it in a follow up?

Reply:
Will deal with this in a future PR and merge this now